core/client/src/client.rs

```diff
@@ -1534,18 +1513,6 @@ pub(crate) mod tests {
 		);
 	}
 
-	#[test]
-	fn authorities_call_works() {
```

Comment:
Should this test be moved to `aura`/`consensus`?

Reply:
yeah perhaps in `consensus`.